docs/log-data-in-reportportal/HotToRunYourTests.md

```diff
@@ -0,0 +1,6 @@
+# Hot to run your tests
+
+Automated tests can be executed in the any CI tool you are using on the project (Jenkins, Bamboo, Azure DevOps, GitHub Actions, CircleCI, TeamCity, GitLab, Travis CI, Codeship, Buddy, GoCD, Wercker).
```

Comment:
(Jenkins, Bamboo, Azure DevOps, GitHub Actions, CircleCI, TeamCity, GitLab, Travis CI, Codeship, Buddy, GoCD, Wercker `and more`).